.github/labels.yml

```diff
@@ -39,3 +39,6 @@ has-e2e-2tests:
 
 has-changelog:
   - "changelogs/**"
+
+changelog-not-required:
```

Comment:
The labeling and changelog checking are two separated jobs, not sure which one will run first. If the changelog checking job runs first, seems the job will still fail

Let's remove the change in this PR first during the RC period, and do more investigation about it later

Reply:
OK，it's removed.